pymchelper/detector.py

```diff
@@ -1,37 +1,37 @@
-from collections import namedtuple, defaultdict
+from collections import namedtuple
+from enum import IntEnum
 import logging
-import os
 
 import numpy as np
-from enum import IntEnum
-
-from pymchelper.readers.fluka import FlukaBinaryReader
-from pymchelper.readers.shieldhit import SHTextReader, SHBinaryReader
-from pymchelper.shieldhit.detector.detector_type import SHDetType
-from pymchelper.shieldhit.detector.estimator_type import SHGeoType
-from pymchelper.writers.excel import ExcelWriter
-from pymchelper.writers.inspector import Inspector
-from pymchelper.writers.plots import ImageWriter, GnuplotDataWriter, PlotDataWriter
-from pymchelper.writers.shieldhit import TxtWriter
-from pymchelper.writers.sparse import SparseWriter
-from pymchelper.writers.trip98 import TripCubeWriter, TripDddWriter
 
 logger = logging.getLogger(__name__)
```

Comment:
I think you never use logger in this file 😉

Reply:
But I will